paywall/data-iframe.2.0.webpack.config.js

```diff
@@ -6,7 +6,7 @@ const configVariables = require('./environment')
 module.exports = () => {
   return {
     cache: false,
-    mode: 'production',
+    mode: 'development',
```

Comment:
I had to switch this temporarily because the browser complained it couldn't find the related source maps. I'm sure there's a configuration setting I have to switch somewhere.